coraza.go

```diff
@@ -182,23 +186,29 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 	return m, err
 }
 
-func newErrorCb(logger *zap.Logger) func(types.MatchedRule) {
+func newErrorCb(logger *zap.Logger, tag string) func(types.MatchedRule) {
 	return func(mr types.MatchedRule) {
 		data := mr.ErrorLog(403)
+		var msg string
+		if tag == "" {
+			msg = data
+		} else {
+			msg = fmt.Sprintf("[tag %s] %s", tag, data)
```

Comment:
The tag prefix interpolation might happen only oncean be referenced in this callback I think.

Reply:
this tag will be happen every time it breaks the rule,I think it's enough for debug